keyboards/planck/ez/ez.c

```diff
@@ -126,11 +126,13 @@ void matrix_scan_kb(void) {
   matrix_scan_user();
 }
 
+#ifdef PLANCK_EZ_CUSTOM_LEDS
```

Comment:
Setting this to `ifndef` may be better behavior.  This way, you don't need to define anything to have this working. 

```suggestion
#ifndef PLANCK_EZ_CUSTOM_LEDS
```